index.d.ts

```diff
@@ -0,0 +1,42 @@
+// https://github.com/Microsoft/TypeScript/blob/9d3707d671592d030386956c9ce39e539b8d0972/src/lib/dom.generated.d.ts#L10581
+
+type baseElements = Element | Element[] | NodeList | DocumentFragment;
+
+export interface SelectDom {
```

Comment:
I think you want to default export this one, so it's imported when you import `select-dom`.

Relevant: https://github.com/Microsoft/TypeScript/issues/3792#issuecomment-303526468